app/assets/javascripts/2017/modules/main-menu.js.coffee

```diff
@@ -45,3 +45,6 @@ $(document).ready ->
     $('body').css('position', 'fixed')
     $('#btn-menu').addClass('active')
     $("#main-menu").css('overflow-y', 'auto')
+
+  $(".ticket-background").click ->
+    window.open("https://www.eventbrite.com/e/magmaconf-2017-evolution-has-come-tickets-31259518060?aff=magmaconfwebsite")
```

Comment:
Why do you use a javascript event to open a page?

Why not just using an anchor? like `<a href="link"></a>`